src/platform_impl/macos/appkit/view.rs

```diff
@@ -59,6 +59,9 @@ extern_methods!(
     }
 
     unsafe impl NSView {
+        #[sel(setNeedsDisplay:)]
+        pub fn setNeedsDisplay(&self, needs_display: bool);
+
```

Comment:
Not sure why this was added, and I don't see it in `NSView` docs in `Appkit`.

Reply:
hmm, yeah, I guess this was a left over mistake from when I wanted to have all `RedrawRequests` go via `drawRect` but apparently I was looking at UIView docs :/